jenkins/ml-models.JenkinsFile

```diff
@@ -28,17 +28,25 @@ pipeline {
         )
         }
     environment {
-        ARTIFACT_PATH = "${BASE_DOWNLOAD_PATH}/${VERSION}/"
+        ARTIFACT_PATH = "${env.BASE_DOWNLOAD_PATH == "ml-models/model_listing" ? "${BASE_DOWNLOAD_PATH}" : "${BASE_DOWNLOAD_PATH}/${VERSION}/"}"
```

Comment:
Did not get why `env.` was added

Reply:
That is actual way in Jenkins to use the environment variables.

Reply:
Yeah my question was where are we adding `BASE_DOWNLOAD_PATH` as env variable. Are parameters by default env variables?

Reply:
Ok here is some information.
`env` contains all environment variables.
`params` contains all build parameters.
Jenkins also automatically creates an environment variable for each build parameter. Hence be accessed with `env.`

Reply:
Thanks that answers it Can you make it consistent across the jenkins file then instead of just one instance? Would help to avoid confusion

Reply:
Sure thanks